<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordsTeacher</class>
 <widget class="QMainWindow" name="WordsTeacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>English Teacher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/images/images/icon.png</normaloff>:/new/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="field_eng">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_pl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reactionButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show answer</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_eng">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ENG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pl">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>PL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="optionsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QGroupBox" name="optionsTranslationDirectionBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Translation direction</string>
     </property>
     <widget class="QRadioButton" name="randomDirection">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rando&amp;m direction</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="plToEng">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>P&amp;L to ENG</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="engToPl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>89</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ENG to PL</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="optionsSelectionOrderBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>231</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Selection order</string>
     </property>
     <widget class="QRadioButton" name="randomOrder">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rando&amp;m</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="sequentialOrder">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>133</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Se&amp;quential</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
